src/mono/wasm/README.md

```diff
@@ -163,11 +163,11 @@ The samples in `src/mono/sample/wasm` can be build and run like this:
 
 * console Hello world sample
 
-`dotnet build /t:RunSample console-v8-cjs/Wasm.Console.V8.CJS.Sample.csproj`
+`dotnet build /t:RunSample console-v8-es6/Wasm.Console.V8.Sample.csproj`
```

Comment:
Does `-es6` need to be removed here?

Reply:
shuffling folders would take more effort, let's do it later.